backend/clj/rehearser/api/exercise.clj

```diff
@@ -15,38 +40,50 @@
     {:status 404
      :body "Not found, not changed"}))
 
+(defn response-update-one [result]
+  (if result
+    {:status 200
+     :body result}
+    {:status 404
+     :body "Not found, not changed"}))
+
 (defn get-exercises [{:keys [db whoami]}]
   {:status 200
    :body (service/find-all db whoami)})
 
-(defn post-exercise! [{:keys [db whoami body-params]
-                      :as req}]
-  {:body (service/add! db whoami body-params)
+(defn post-exercise! [{{:keys [body]} :parameters
```

Comment:
Regardless of which validation framework is used, the functions need to be migrated to look in `:parameters`.